package.json

```diff
@@ -2,7 +2,21 @@
   "name": "mantra-demo",
   "version": "0.0.0",
   "dependencies": {
-  
+  },
+  "eslintConfig": {
+    "extends": "eslint:recommended",
```

Comment:
I prefer to use an external .eslintrc file for this. 
Could you pick it from our npm-base repo. I am on mobile, couldn't find the link. 
Sorry about that.